static/app/components/group/issueReplayCount.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from '@emotion/styled';
+
+import Link from 'sentry/components/links/link';
+import {useReplaysCountContext} from 'sentry/components/replays/replayCountContext';
+import Tooltip from 'sentry/components/tooltip';
+import {IconPlay} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import useOrganization from 'sentry/utils/useOrganization';
+
+type Props = {
+  groupId: string;
+};
+
+/**
+ * Show the count of how many replays are associated to an issue.
+ */
+function IssueReplayCount({groupId}: Props) {
+  const organization = useOrganization();
+  const count = useReplaysCountContext()[groupId];
+
+  if (count === undefined) {
+    return null;
+  }
+
+  const countDisplay = count > 50 ? '50+' : count;
```

Comment:
Isn't default limit == 50? If that's the case, we'll never have > 50.

*edit* nm, I see that we changed the query so that it's aggregated in the backend rather than counting rows of the result.